registry/purview-registry/main.py

```diff
@@ -43,6 +44,48 @@ def to_camel(s):
                    allow_headers=["*"],
                    )
 
+def exc_to_content(e: Exception) -> dict:
+    content={"message": str(e)}
+    if os.environ.get("REGISTRY_DEBUGGING"):
```

Comment:
Should we document all the envs somewhere?

Reply:
Good catch, doc is updated